airbyte-cdk/python/airbyte_cdk/sources/declarative/stream_slicers/datetime_stream_slicer.py

```diff
@@ -75,8 +75,8 @@ def __post_init__(self, parameters: Mapping[str, Any]):
         self._cursor_granularity = self._parse_timedelta(self.cursor_granularity)
         self.cursor_field = InterpolatedString.create(self.cursor_field, parameters=parameters)
         self.lookback_window = InterpolatedString.create(self.lookback_window, parameters=parameters)
-        self.stream_slice_field_start = InterpolatedString.create(self.stream_state_field_start or "start_time", parameters=parameters)
-        self.stream_slice_field_end = InterpolatedString.create(self.stream_state_field_end or "end_time", parameters=parameters)
+        self.partition_field_start = InterpolatedString.create(self.stream_state_field_start or "start_time", parameters=parameters)
```

Comment:
as per over the shoulder, we still need to move the datetimestreamslicer so we can rename partitions then